package.json

```diff
@@ -78,6 +83,7 @@
     "targz": "1.0.1",
     "uglify-js": "2.6.4",
     "underscore": "1.8.3",
-    "watch": "0.19.1"
+    "watch": "0.19.1",
+    "webpack": "1.14.0"
```

Comment:
Worth upgrading to 2.0 since the new hot sauce has been released... Maybe too soon?